### PR TITLE
doc review for vol-plugin

### PR DESCRIPTION
**Pull request by vvb**
(1) git clone command was not complete and executable
(2) formatting was off at multiple places which wasn't showing the bullet count correctly
(3) the port that volplugin uses to connect to volmaster was specified incorrectly.

### CODE REVIEW

File: source/2_volplugin.md [hunk 1]
```diff
@@ -44,20 +44,20 @@ Ensure ceph is fully operational, and that the `rbd` tool works as root.
...
-    * `cat systemtests/testdata/intent1.json | volcli tenant upload tenant1`
+    * If you just want a quick start without configuring it yourself: 
+        * `cat systemtests/testdata/intent1.json | volcli tenant upload tenant1`
 1. Start volmaster in debug mode (as root): `volmaster --debug &`
-  * volmaster has a debug mode as well, but it's really noisy, so avoid using
-    it with background processes. Volplugin currently runs on port 8080, but
-    this will be variable in the future.
+    * volmaster has a debug mode as well, but it's really noisy, so avoid using
+    it with background processes. volplugin currently connects to volmaster
+    using port 9095, but this will be variable in the future.
```
Comment (erikh) [inline source/2_volplugin.md]: 9005

Comment (vvb) [inline source/2_volplugin.md]: @erikh thanks for catching that.

File: source/2_volplugin.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ before attempting these instructions.
 
 Clone and build the project: 
 
-* `git clone github.com/contiv/volplugin`
+* `git clone https://github.com/contiv/volplugin.git`
```
Comment (erikh) [inline source/2_volplugin.md]: no `.git` at the end

File: source/2_volplugin.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Clone and build the project:
 
 #### Everywhere else (with a VM):
 
-* `git clone github.com/contiv/volplugin`
+* `git clone https://github.com/contiv/volplugin.git`
```
Comment (erikh) [inline source/2_volplugin.md]: same here, no `.git`

**Comment by erikh** [comment]
LGTM other than the above comments.

File: source/2_volplugin.md (hunk 3 above)
Comment (vvb) [inline source/2_volplugin.md]: oh, I have always used a ".git". But just checked, that it works without it too. I will remove the ".git" Thanks.

**Comment by vvb** [comment]
ping @erikh I have made modifications as per your comments. PTAL

**Pull request merged**